### PR TITLE
mes-8942-MOTAnalytics

### PR DESCRIPTION
**Pull request by RLCorp**
## Description
Added Analytics to MOT
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Review by gittins7 — APPROVED** [review]

File: src/app/pages/waiting-room-to-car/waiting-room-to-car.analytics.effects.ts [hunk 1]
```diff
@@ -470,12 +483,259 @@ export class WaitingRoomToCarAnalyticsEffects {
...
+          withLatestFrom(this.store$.pipe(select(getTests)), this.store$.pipe(select(getTests), select(isPracticeMode)))
+        )
+      ),
+      filter(([, , practiceMode]) =>
+        !practiceMode ? true : this.appConfigProvider.getAppConfig()?.journal?.enablePracticeModeAnalytics
+      ),
+      switchMap(([{ method }, tests]: [ReturnType<typeof MotCallAborted>, TestsModel, boolean]) => {
+        let abortMethod = GoogleAnalyticsEventsValues.UNKNOWN;
+
+        switch (method) {
```
Comment (andrewsetterfield) [inline src/app/pages/waiting-room-to-car/waiting-room-to-car.analytics.effects.ts:512]: a switch without a default is bad practice.  suggest that a setting to GoogleAnalyticsEventsValues.UNKNOWN is handled via a default

switch (method) {
  case MOTAbortedMethod.END_TEST:
    abortMethod = GoogleAnalyticsEventsValues.END_TEST_SELECTED;
    break;
  case MOTAbortedMethod.VRN_CHANGED:
    abortMethod = GoogleAnalyticsEventsValues.VRN_EDITED;
    break;
  case MOTAbortedMethod.NAVIGATION:
    abortMethod = GoogleAnalyticsEventsValues.NAVIGATION;
    break;
  default:
    abortMethod = GoogleAnalyticsEventsValues.UNKNOWN;
}

File: src/app/providers/analytics/analytics.model.ts [hunk 2]
```diff
@@ -351,6 +351,12 @@ export enum GoogleAnalyticsEvents {
 }
 
 export enum GoogleAnalyticsEventsTitles {
+  API_UNAVAILABLE = 'api_unavailable',
+  CHECK_CANCELLED = 'check_cancelled',
+  MOT_WARNING = 'mot_warning',
+  ALT_EVIDENCE_PROVIDED = 'alt evidence provided',
```
Comment (andrewsetterfield) [inline src/app/providers/analytics/analytics.model.ts]: triggers me that this doesnt follow the pattern.  please add underscores

Comment (RLCorp) [reply src/app/providers/analytics/analytics.model.ts]: fixed in newest commit

File: src/app/pages/waiting-room-to-car/waiting-room-to-car.analytics.effects.ts (hunk 1 above)
Comment (RLCorp) [reply src/app/pages/waiting-room-to-car/waiting-room-to-car.analytics.effects.ts:512]: fixed in newest commit

**Review by andrewsetterfield — APPROVED** [review]

**Pull request merged**